### PR TITLE
New package: bibletime-2.11.2

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

File: srcpkgs/bibletime/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ qt5-svg-devel qt5-webkit-devel sword-devel zlib-devel"
+short_desc="Powerful cross platform Bible study tool"
+maintainer="fosslinux <fosslinux@aussies.space>"
+license="GPL-2.0-or-later"
+homepage="http://bibletime.io"
+distfiles="https://github.com/bibletime/bibletime/releases/download/v${version}/bibletime-${version}.tar.xz"
+checksum=d178197ce87c8211733409fb28fa6ed0220800bbf06944ebb8fe85bf78ddabe8
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-host-tools qt5-qmake"
```
Comment (Chocimier) [inline srcpkgs/bibletime/template:20]: Add unconditionally.

**Comment by fosslinux** [comment]
Oh crap I forgot to do tests!! Thanks

**Comment by Chocimier** [comment]
Ping?

**Comment by fosslinux** [comment]
Sorry, I've added the block but tests need fixing on i686. I've urgently needed to get something else fixed over the last few days, but I should get back to it today

**Comment by fosslinux** [comment]
I have no clue what the problem is with the tests on i686. It seem to be some strange, isolated locale issue.

**Comment by fosslinux** [comment]
Same thing on x86_64 now?!

```
FAIL!  : test_CSwordBackend::booknameLanguage(L2) Compared values are not the same
   Actual   (QString(vKey->getBookName())): "John"
   Expected (bookName)                    : "Johannes"
   Loc: [/builddir/bibletime-2.11.2/tests/backend/managers/cswordbackend/test_cswordbackend.cpp(236)]
PASS   : test_CSwordBackend::booknameLanguage(L3)
FAIL!  : test_CSwordBackend::booknameLanguage(L4) Compared values are not the same
   Actual   (QString(vKey->getBookName())): "Genesis"
   Expected (bookName)                    : "1. Mose"
   Loc: [/builddir/bibletime-2.11.2/tests/backend/managers/cswordbackend/test_cswordbackend.cpp(236)]
```

**Comment by fosslinux** [comment]
https://github.com/bibletime/bibletime/issues/259 :

> I would currently discourage you to run the tests at all.

Hence, I don't think this should be a blocking failure. Although, I will try what they said:

> using `ctest` seems to work properly for me, with all 5 tests passing.

**Comment by Chocimier** [comment]
Tests work for me, is your locale leaking into chroot? Maybe try calling env(1) in pre_check to see it.

**Comment by fosslinux** [comment]
I bet it is; and if so, that's an xbps-src bug. I will continue to test this, so I can find where the issue lies.

**Pull request merged**